app/models/organization.rb

```diff
@@ -227,7 +227,7 @@ def current_invoices; invoices.active end
   def current_parent_invoices; Invoice.where(organization_id: parent_organization_id).active end
 
   def incomplete_b_params
-    BParam.where(organization_id: child_ids + [id]).partial_registrations.without_bike
+    BParam.where(organization_id: [*child_ids, id]).partial_registrations.without_bike
```

Comment:
Is this better than using flatten? I feel like the splat is surprising, but maybe that's just me

Reply:
I'm used to it from seeing the pattern in JS, but it might be on the terse side for Ruby — updated in ce440a08